app/controllers/ApplicationController.java

```diff
@@ -40,6 +45,24 @@
     @Inject
     public ApplicationController(final Features features) {
         _features = features;
+        _featuresJson = Suppliers.memoize(() -> {
+            try {
+                return OBJECT_MAPPER.writeValueAsString(_features);
+            } catch (final JsonProcessingException e) {
+                throw new RuntimeException(e);
+            }
+        });
+    }
+
+    /**
+     * Render configure javascript.
+     *
+     * @return Rendered configure javascript.
+     */
+    public CompletionStage<Result> getConfigureJs() {
+        return CompletableFuture.completedFuture(
+                ok(views.html.ConfigureJsViewModel.render(_featuresJson.get()))
```

Comment:
No, I'm not a fan. The alternative was to do the json serialization inside the template passing in the object and an ObjectMapper. I'm not sure that's any better, especially since it's the same content over and over again.